clang/lib/Basic/Targets/AArch64.h

```diff
@@ -32,6 +58,8 @@ class LLVM_LIBRARY_VISIBILITY AArch64TargetInfo : public TargetInfo {
     SveMode = (1 << 2),
   };
 
+  enum AddrSpace { ptr32_sptr = 270, ptr32_uptr = 271, ptr64 = 272 };
```

Comment:
This is kind of nitty, but if you move this up to global scope and give it an AArch64-prefixed name you can use `static_cast<unsigned>(AArch64::ptr32_sptr)` in place for 270 etc above.

Reply:
Definitely prefer that - this whole thing has **way** too many magic numbers for my liking...